ansible/configs/test-empty-config/post_software.yml

```diff
@@ -22,9 +22,9 @@
       pause:
         seconds: 30
 
-    - debug:
+    - agnosticd_user_info:
         msg: >-
-          user.info: Some random password
+          Some random string
```

Comment:
Note to myself: update https://github.com/redhat-cop/babylon/blob/master/test/babylon-empty-config.groovy#L132